com.unity.netcode.gameobjects/Runtime/Messaging/MessageQueue/MessageQueueHistoryFrame.cs

```diff
@@ -78,7 +78,10 @@ internal MessageFrameItem GetCurrentQueueItem()
             m_CurrentItem.MessageType = (MessageQueueContainer.MessageType)QueueReader.ReadUInt16();
             m_CurrentItem.Timestamp = QueueReader.ReadSingle();
             m_CurrentItem.NetworkId = QueueReader.ReadUInt64();
-            m_CurrentItem.NetworkChannel = (NetworkChannel)QueueReader.ReadByteDirect();
+            if (m_QueueFrameType == QueueFrameType.Outbound)
+            {
+                m_CurrentItem.Delivery = (NetworkDelivery)QueueReader.ReadByteDirect();
```

Comment:
hrm.

Reply:
we carry `NetworkDelivery` around for outgoing packets but not for incoming packets.
having said that, we'll probably rework message queuing/batching etc later on when we integrate RPCs, snapshotted NetVars etc.

Reply:
Yeah, I know it just always reads weird to me every time I see it :)